### DISCUSSION TITLE
Newbie question

### DISCUSSION THREAD

**Discussion by Apollo355** [community/community#55947] [New to GitHub] [answered]
Hi guys, first time poster on Github here. :) How do I load codes onto my repositories?

**Comment by K1ethoang** [comment]
Did you have create a repo?

**Reply by Apollo355** [reply]
No.  Just need to know how.

**Comment by Apollo355** [comment]
Previously, I've had developers upload them, but now like to know how to do it myself.  So any help would be greatly appreciated!

**Comment by airtower-luna** [comment, marked as answer]
You need to use Git to work with the repositories. Check out the [Git Book](https://git-scm.com/book/en/v2) (it's a website, though you can download an ebook). The first three chapters cover the basics, including what Git is. :slightly_smiling_face:

**Comment by RAPANALINARIVO** [comment]
fatal: unable to access 'https://github.com/RAPANALINARIVO/SAYNA-HTMLCSS-BATMAN1.git/': Failed to connect to github.com port 443 after 76 ms: Couldn't connect to server
need help , how can i resolve it pls?

**Reply by airtower-luna** [reply]
The error message looks like a network problem. It's a completely different topic, though.

**Comment by Zippyflash62** [comment]
Im not going to click that link. Sorry but I need help learnïng. Myself. I apoligize to the community.

**Comment by 1ANSAH** [comment]
I am not receieving by Github payment bill

**Comment by Nurse-Dee1** [comment]
Hello guys, I was trying to deploy on vercel, but vercel wouldn't finish with the authorization. It says user has been blocked. How do I go about this
